ui/pages/send/send-content/send-asset-row/send-asset-row.component.js

```diff
@@ -75,28 +109,46 @@ export default class SendAssetRow extends Component {
     return (
       <SendRowWrapper label={`${t('asset')}:`}>
         <div className="send-v2__asset-dropdown">
-          {this.renderSendToken()}
-          {this.state.sendableTokens.length > 0
+          <div
+            className="send-v2__asset-dropdown__input-wrapper"
+            onClick={this.openDropdown}
+          >
+            {this.renderSendAsset()}
+          </div>
+          {[...this.state.sendableTokens, ...this.state.sendableCollectibles]
+            .length > 0
             ? this.renderAssetDropdown()
             : null}
         </div>
       </SendRowWrapper>
     );
   }
 
-  renderSendToken() {
-    const { sendAssetAddress } = this.props;
-    const token = this.props.tokens.find(({ address }) =>
-      isEqualCaseInsensitive(address, sendAssetAddress),
-    );
-    return (
-      <div
-        className="send-v2__asset-dropdown__input-wrapper"
-        onClick={this.openDropdown}
-      >
-        {token ? this.renderAsset(token) : this.renderNativeCurrency()}
-      </div>
-    );
+  renderSendAsset() {
+    const {
+      sendAsset: { details, type },
+      tokens,
+      collectibles,
+    } = this.props;
+
+    if (type === 'TOKEN') {
```

Comment:
Is it worth it to extract these strings to constants?

Reply:
As a matter of fact these constants already exist on an `ASSET_TYPES` enum. Will swap out for this.

Reply:
done here: https://github.com/MetaMask/metamask-extension/pull/13048/commits/e0a1b4c80da158cc529475d6cdf7faca39d269d6